etl/glean_auto_events.py

```diff
@@ -23,9 +23,12 @@ def _query_bigquery(query):
 
 def _get_auto_events_names(app):
     """Get the automatic events names for the app"""
-    query = f"""SELECT *
-                FROM `moz-fx-data-shared-prod.glean_dictionary_derived.auto_events_metadata`
-                WHERE app = '{app}' """
+    query = f"""SELECT
+                    app,
+                    name
+                FROM `moz-fx-data-shared-prod.glean_dictionary_derived.auto_events_metadata_v1`
+                WHERE app = '{app}'
```

Comment:
Should this also filter by `submission_date`?

Reply:
Not really, we want distinct pairs of (app, name) regardless of date. The table is clustered on those columns so I think it'll be fine

Reply:
I see a risk of some "trash" data surfacing here and staying forever. I haven't checked if this does happen, but we could for example limit this to last 30 days or use some count threshold. Can be changed later though, I agree.

Re. sanitization, I mentioned some in another PR: https://github.com/mozilla/bigquery-etl/pull/6987#pullrequestreview-2609373141